mlflow/tracking/client.py

```diff
@@ -460,7 +460,11 @@ def search_registered_models(
         """
         Search for registered models in backend that satisfy the filter criteria.
 
-        :param filter_string: Filter query string, defaults to searching all registered models.
+        :param filter_string: Filter query string, defaults to searching all registered
+                models. Currently, it supports only a single filter condition as the name
+                of the model, for example, ``name = 'model_name'`` or a search expression
+                to match a pattern in the registered model name.
+                For example, ``name LIKE 'Boston%'`` or ``name ILIKE '%Forecast%'``.
```

Comment:
```suggestion
                For example, ``name LIKE 'Boston%'`` (case sensitive) or ``name ILIKE '%Forecast%'`` (case insensitive).
```

I think users who are not familiar with SQL would not understand what `ILIKE` means.

Reply:
Indeed, make it more specific, as the API does not describe it. I'll add this too.